app/design/frontend/Magento/elogic-theme/Magento_Cms/web/css/source/_module.less

```diff
@@ -281,6 +292,65 @@
         line-height: 1.5;
     }
 
+    // SHOP BY BRANDS
+
+    .shop-by-brands {
+        margin-top: @indent__l*2;
+        margin-bottom: @indent__m*4;
+    }
+
+    .brands-title {
+        margin-top: @indent__xl;
+        margin-bottom: @indent__l;
+        text-align: center;
+        overflow: hidden;
+        strong {
+            .lib-css(color, @color-elogic-black);
+            .lib-css(font-size, @font-size__xl_xl);
+            .lib-css(font-weight, @font-weight-regular);
+            margin-bottom: @indent__s;
+            padding: 0;
+            text-transform: capitalize;
+            position: relative;
+            display: inline-block;
+            line-height: 30px;
+            padding: 0 @indent__xs;
+            &::before {
+                content: "";
+                display: block;
+                width: 1168px;
```

Comment:
Does this width work correctly for all screen sizes? Maybe it's better to change it to 100%?